cmake/Modules/Dependencies.cmake

```diff
@@ -33,13 +33,22 @@ if(BUILD_DOCS)
     set(rocm_cmake_tag
       "c044bb52ba85058d28afe2313be98d9fed02e293" # develop@2023.09.12. (move to 6.0 tag when released)
       CACHE STRING "rocm-cmake tag to download")
+
+    # Define the cache directory
+    set(FETCHCONTENT_BASE_DIR "${CMAKE_BINARY_DIR}/_deps_cache")  
+
```

Comment:
According to cmake documentation the default for FETCHCONTENT_BASE_DIR is `${CMAKE_BINARY_DIR}/_deps`

Why does changing the name to `${CMAKE_BINARY_DIR}/_deps_cache` help?

https://cmake.org/cmake/help/latest/module/FetchContent.html